src/oci.c

```diff
@@ -619,132 +619,6 @@ set_env_home(struct cc_oci_config *config)
 	config->oci.process.env = new_env;
 }
 
-
-/*!
- * Create the containers Clear Linux workload file
- * (\ref CC_OCI_WORKLOAD_FILE).
- *
- * \param config \ref cc_oci_config.
- *
- * \warning FIXME: Need to support running the workload as a different user/group.
- * Something like:
- *
- *      su -c "sg $group -c $cmd" $user
- *
- * \return \c true on success, else \c false.
- */
-private gboolean
-cc_oci_create_container_workload (struct cc_oci_config *config)
-{
-	GString           *contents = NULL;
-	GError            *err = NULL;
-	gchar             *workload_cmdline = NULL;
-	g_autofree gchar  *path = NULL;
-	g_autofree gchar   *cwd = NULL;
-	gboolean           ret = false;
-	gchar             **args = NULL;
-
-	if (! (config && config->oci.process.args)) {
-		return false;
-	}
-
-	if (! config->vm) {
-		g_critical ("No vm configuration");
-		goto out;
-	}
-
-	if (config->oci.process.env) {
-		g_autofree gchar  *envpath = NULL;
-		g_autofree gchar  *env = NULL;
-
-		envpath = g_build_path ("/", config->oci.root.path,
-				CC_OCI_ENV_FILE, NULL);
-		if (! envpath) {
-			return false;
-		}
-
-		set_env_home(config);
```

Comment:
This function can be dropped now as it is unused (along with the `test_set_env_home()`).